build.gradle

```diff
@@ -23,10 +23,15 @@ buildscript {
 }
 
 plugins {
+    //The version number is duplicated in the detekt block. We are unable to use constants due to
```

Comment:
nit: There should be a space between the comment markers and the comments, e.g. `// The version number...`

Reply:
Done :)